tao_compiler/mlir/disc/transforms/tests/common_rewriter.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: disc-opt -disc-common-rewriter -split-input-file %s -o - | FileCheck %s
+
+// CHECK-LABEL: func.func @two_side_reduction
+func.func @two_side_reduction(%arg0: tensor<?x?x?x?xf16>) -> tensor<?xf16> {
+  %0 = mhlo.constant {disc.device = "gpu"} dense<0.000000e+00> : tensor<f16>
+  // CHECK: mhlo.reduce
```

Comment:
please check row/column reduce ops have the right `dimension` attrs.

Reply:
The dimensions check is split into another filecheck ut, see `two_side_reduction_check_dims`

Reply:
ok. maybe be just merge these two tests using `CHECK-SAME`?